src/validate.ts

```diff
@@ -8,7 +8,7 @@ export class InvalidTransitionCondition extends Error {}
 export function validateStates<TContext, TEvent>(
 	states: Array<State<TContext, TEvent>>,
 ) {
-	if (states?.length !== 2)
```

Comment:
@marco-ippolito i think that the statement `states?.length !== 2` was only an old hardcoded value

Reply:
Yes that was an error